src/scancode/cli.py

```diff
@@ -185,6 +186,11 @@ def validate_formats(ctx, param, value):
         raise click.BadParameter('Invalid template file: "%(value)s" does not exists or is not readable.' % locals())
     return value
 
+def ignore_files(ctx, param, value):
+    for file in value.split(','):
+        user_ignored.update({
```

Comment:
How about I define a callback in the ignore.py https://github.com/nexB/scancode-toolkit/blob/develop/src/commoncode/ignore.py, which can modify a dict in the same file and can be later imported through this function https://github.com/nexB/scancode-toolkit/blob/develop/src/scancode/cli.py#L521
Would this be a viable solution to **not** defining globals?
This will also